.ci/pipelines/pr-gke.Jenkinsfile

```diff
@@ -71,7 +71,7 @@ pipeline {
                         }
                     }
                     steps {
-                        sh '.ci/setenvconfig pr'
+                        sh '.ci/setenvconfig pr kindest/node:v1.22.0@sha256:b8bda84bb3a190e6e028b1760d277454a72267a5454b57db34437c34a588d047 0.11.1 ipv4'
```

Comment:
Should we just use/update the defaults in the `kind-ci` item in  `hack/deployer/config/plans.yaml` instead of passing the arguments here?

Reply:
Good idea.